### DISCUSSION TITLE
1.3.4

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#244] [Обновления]
## What's Changed
* Перевод на родной язык by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/217
* Замена библиотеки на сайт (INF DEV) by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/227

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.3.3...1.3.4

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.3.4'>1.3.4</a>.</em>

**Comment by Balzak1976** [comment]
Я и заметил, стало моментально переводить, :muscle:
a то раньше столько ждать приходилось 👍

**Comment by narimangafurov** [comment]
Спасибо автору за развитие инструмента, это многим будет мега полезно! но я лично устал бороться со скачками звука, когда оригинал прыгает все время( в старых версиях такого не было) что жутко неприятно( психанул, поставил рядом Яндекс браузер )) чисто для видео сервисов, который летает и все работает отлично! всем успехов! )
